### PR TITLE
Make the good text be green in build results.

### PR DESCRIPTION
**Pull request by koddsson**
Huge fan of dpxdt at work but I thought the 'Good' build status could use a little color!

Here's a screenshot:
screen shot 2015-01-20 at 17 59 35

### CODE REVIEW

File: dpxdt/server/templates/view_build.html [hunk 1]
```diff
@@ -65,17 +65,10 @@
...
-                                                {{ candidate.status|title }}
-                                            </strong>
-                                        {% elif candidate.status == 'bad' %}
-                                            <span class="text-error">
-                                                {{ candidate.status|title }}
-                                            </span>
-                                        {% else %}
+                                       {% set class_name = 'text-success' if candidate.status == 'good' else 'text-error' %}
+                                        <span class='{{ class_name }}'>
                                             {{ candidate.status|title }}
```
Comment (bslatkin) [inline dpxdt/server/templates/view_build.html]: In the case the status isn't `good` or `bad` (so: processing) the old template will just show the text plain. This change makes it so the processing state will now show as an error. So maybe there's a smaller change we can make to this template to make the good state green? I figure it's replacing the `<strong>` tag above with a span that has a class of `text-success`.

**Comment by bslatkin** [comment]
Thanks! And thanks for doing this. I had one question for you above.

**Comment by koddsson** [comment]
I'd forgotten that there were more statuses than just "good" or "bad" when I did this patch :yum:

I've update it like you suggested @bslatkin

**Comment by bslatkin** [comment]
Hey @elsigh does this look good to you? Should we merge up?

**Comment by elsigh** [comment]
LGTM =)

**Pull request merged**